synthtool/gcp/templates/python_library/README.rst

```diff
@@ -15,7 +15,7 @@ Python Client for {{ metadata['repo']['name_pretty'] }} API
 .. |versions| image:: https://img.shields.io/pypi/pyversions/{{ metadata['repo']['distribution_name'] }}.svg
    :target: https://pypi.org/project/{{ metadata['repo']['distribution_name'] }}/
 .. _{{ metadata['repo']['name_pretty'] }} API: {{ metadata['repo']['product_documentation'] }}
-.. _Client Library Documentation: {{ metadata['repo']['client_documentation'] }}
+.. _Client Library Documentation: {{ metadata['repo']['client_documentation'] }}/summary_overview
```

Comment:
Should we add a conditional check to only append `/summary_overview` if `{{ metadata['repo']['client_documentation'] }}` contains `cloud.google.com`?